collapsiblecalendarview2/src/main/java/com/hyperexternal/collapsiblecalendarview/data/CalendarAdapter.kt

```diff
@@ -126,16 +125,6 @@ class CalendarAdapter(val context: Context, cal: Calendar) {
 
             val view = mInflater.inflate(R.layout.day_layout, null)
             val txtDay = view.findViewById<View>(R.id.txt_day) as TextView
-            val parent = txtDay.getParent() as View?
-            parent?.post {
-                val r = Rect()
-                txtDay.getHitRect(r)
-                r.top -= context.dipToPixels(20).toInt()
-                r.bottom += context.dipToPixels(20).toInt()
-                r.left -= context.dipToPixels(20).toInt()
-                r.right += context.dipToPixels(20).toInt()
-                parent.touchDelegate = TouchDelegate(r, txtDay)
-            }
```

Comment:
It seems this is just removing the touch area rather than increasing the size. How is the touch target size being increased by removing this?

Reply:
this code was written by kgoel in an attempt to increase the touch targets of the days of the calendar but it doesnt work so I am removing it. It doesnt work because each day the touch target is already as big as it can be and all the days of the month are next to each other so increasing touch target size just makes them all overlap each other accomplishing nothing